tests/integration_tests/cyclic_abc/run.sh

```diff
@@ -1,190 +0,0 @@
-#!/bin/bash
-
-set -eu
-
-CUR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
-source $CUR/../_utils/test_prepare
-WORK_DIR=$OUT_DIR/$TEST_NAME
-CDC_BINARY=cdc.test
-SINK_TYPE=$1
-TLS_DIR=$(cd $CUR/../_certificates && pwd)
-
-function run() {
-	# kafka is not supported yet.
-	if [ "$SINK_TYPE" == "kafka" ]; then
-		return
-	fi
-
-	rm -rf $WORK_DIR && mkdir -p $WORK_DIR
-
-	start_tidb_cluster --workdir $WORK_DIR
-	start_tls_tidb_cluster --workdir $WORK_DIR --tlsdir $TLS_DIR
-
-	echo " \
-[security]
```

Comment:
It contains extra TLS tests. Could you extract the TLS tests?

Reply:
http_api integration contains TLS tests too